doc/mainpage.dox

```diff
@@ -0,0 +1,66 @@
+// -*- lsst-c++ -*-
+
+namespace lsst { namespace meas { namespace extensions { namespace ngmix {
+/**
+@mainpage lsst::meas::extensions::ngmix;  plugins using NGMIX fitters
+This package provides wrappers around certain algorithms in Eric Sheldon's NGMIX package.
+The wrappers allow those algorithms to be accessed from meas_base plugins, and provides
+the standard capabilities for SingleFrameMeasurement, including:
+
+- Initializing the schema of the output SourceCatalog.
+- Providing a measure() method for measuring each input exposure.
+- Writing the measured result to the measRecord provided.
+- Handling errors or unexpected results by setting flags in measRecord.
+
+The NGMIX algorithms do multiple Gaussian fits and return a Gaussian Mixture (GMix).
+The "GMix" object is an additive mixture of Gaussian profiles which can be used
+to model a source or PSF. It will also estimate source ellipticity.
+
+@section ngmixEMPsfApprox Using EMPsfApprox
+Shape measurement requires a estimate of the PSF for each measured object.
+EMPsfApprox is a wrapper around the NGMIX Expectation-Maximization fitter, and is used
+to model the PSF as a GMix. This model can have up to 3 Gaussians, as specified
+with the configuration parameter "nGauss".
+
+The components are stored in the measRecord as multiple lsst.shapelet.ShapeletFunction
+components, and can be fetched from the lsst.afw.table.SourceCatalog using a functor
+key of type lsst.shapelet.MultipleShapeletFunctionKey, as shown below:
+@code{.py}
+    name = "meas_extensions_ngmix_EMPsfApprox"
+    msfKey = MultiShapeletFunctionKey(schema[name], lsst.shapelet.HERMITE)
+    multiShapeletFunction = measRecord.get(msfKey)
+@endcode
+
+To set the number of Gaussians modeled by EMPsfApprox, set the config for the plugin
```

Comment:
Try not to repeat "set" here. Possibly the comma and everything afterwards could be replaced with a colon.